tests/Unit/AuthTokenTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+use Payrexx\Models\Request\AuthToken;
+use Payrexx\PayrexxException;
+
+it('Success response - 200', function () {
+    $mockCommunicator = new class () {
+        public function requestApi(): array
+        {
+            return getMockResponse(
+                200,
+                [
+                    ['authToken' => 'fake-token'],
+                ],
+                'Test Message'
+            );
+        }
+    };
+    $custom = new \Payrexx\Payrexx('dummy', 'dummy', $mockCommunicator::class);
+    $authToken = new AuthToken();
+    $authToken->setUserId('1');
+
+    $response = $custom->create($authToken);
+    expect($response->getAuthToken())->toBe('fake-token');
```

Comment:
Please also check for correct class
```suggestion
    expect($response->getAuthToken())->toBeString('fake-token')
        ->and($response)->toBeInstanceOf(AuthTokenResponse::class);
```